### ISSUE TITLE
Saying "yes" or "no" to resume playing results in error. 

### ISSUE THREAD

**Issue by zallanx** [alexa/skill-sample-nodejs-audio-player#79]
I am trying to run the standard multiple-stream audio player Skill. After starting a podcast, when trying to resume, I get the following error:

![image](https://user-images.githubusercontent.com/1867934/36695108-478b665e-1b0e-11e8-8e3e-038a3d643886.png)

Specifically, a normal "yes", "no", "resume", "stop" or any other commands as suggested from [official documentation](https://developer.amazon.com/docs/custom-skills/standard-built-in-intents.html) would not let me proceed. Is there something I am missing or are there intents that were not implemented that I needed to implement?

Help is much appreciated.

**Comment by sebsto** [comment]
I just downloaded the code and created the skill.  It works as expected.
<img width="407" alt="screen shot 2018-03-11 at 08 53 41" src="https://user-images.githubusercontent.com/401798/37251542-ded7f2b6-2509-11e8-9f01-e94608e7d8ec.png">

I would suggest to log requests and responses sent to the skill and look at AWS Cloudwatch logs to understand what is not working in your code.  I added [code](https://github.com/alexa/skill-sample-nodejs-audio-player/blob/mainline/multiple-streams/lambda/src/index.js#L20)  in ```index.js``` to log the full response, in addition of the request.
